ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 is-land
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import * as generate from 'utils/generate';
+import InputField from '../InputField';
+
+const setup = (override = {}) => {
+  return {
+    input: {
+      name: generate.name(),
+      onChange: jest.fn(),
+      value: generate.name(),
+      //checked: true,
+    },
+    meta: {
+      touched: true,
+      error: generate.message(),
+    },
+    testId: 'inputField',
+    ...override,
+  };
+};
+//helperText: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+//errorMessage: PropTypes.string,
+
+describe('<InputField />', () => {
+  afterEach(() => {
+    cleanup();
+    jest.clearAllMocks();
+  });
+
+  it('renders inputField', () => {
+    const props = setup();
+    render(<InputField {...props} />);
```

Comment:
You can also make this a one-liner like:

```suggestion
    render(<InputField {...setup()} />);
```

Reply:
Wow ! Got it.